build.gradle

```diff
@@ -41,7 +42,7 @@ tasks.shadowJar.dependsOn("myZip");
 tasks.run.dependsOn("myZip");
 
 configurations {
-  functionalCompile.extendsFrom testCompile
+  functionalImplementation.extendsFrom testImplementation
```

Comment:
A side comment: We should use a terms such as 'System Tests' instead of 'functional'.

Reply:
Opened issue #85